e2e/testdata/fn-render/fnconfig-updated-in-render/README.md

```diff
@@ -0,0 +1,21 @@
+# app-example
```

Comment:
It seems this file is just a template without much useful information. Should we remove it?

Reply:
Removed it.